api/run.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import uvicorn
+
+
+if __name__ == "__main__":
+    uvicorn.run(
```

Comment:
Should we explicitly set [`workers=1`](https://www.uvicorn.org/settings/#production) here (even though that's the default)?

Reply:
Sure!